File: app/src/main/java/com/bubelov/coins/api/rates/CoinbaseResponse.java
package com.bubelov.coins.api.rates;

import java.util.Map;

/**
 * @author Igor Bubelov
 */

public class CoinbaseResponse {
    public CoinbaseResponseData data;

    public class CoinbaseResponseData {
        public String currency;

        public Map<String, Double> rates;
    }
}

File: app/src/main/java/com/bubelov/coins/api/rates/CoinbaseResponse.kt
package com.bubelov.coins.api.rates

/**
 * @author Igor Bubelov
 */

class CoinbaseResponse {
    var data: CoinbaseResponseData? = null

    inner class CoinbaseResponseData {
        var currency: String? = null
        var rates: Map<String, Double>? = null
    }
}